INFO     core.jenkins_utils:jenkins_utils.py:210 running cleanup
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 403 None
DEBUG    core.jenkins_utils:jenkins_utils.py:92 failed to get page on the first try, retrying after login
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /login?from=%2F HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /j_spring_security_check HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/securityRealm/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /computer/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /manage/credentials/store/system/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET / HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /me/my-views/view/all/submitDescription HTTP/1.1" 302 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /me/my-views/view/all/ HTTP/1.1" 200 None
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "POST /user/admin/appearance/configSubmit HTTP/1.1" 200 133
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /user/admin/security/ HTTP/1.1" 200 None
DEBUG    conftest:conftest.py:60 Argument --remote-allow-origins=* added to the chrome
DEBUG    conftest:conftest.py:60 Argument --disable-gpu added to the chrome
DEBUG    conftest:conftest.py:60 Argument --no-sandbox added to the chrome
DEBUG    conftest:conftest.py:60 Argument --disable-dev-shm-usage added to the chrome
DEBUG    conftest:conftest.py:60 Argument --headless=new added to the chrome
DEBUG    conftest:conftest.py:60 Argument --window-size=1920,1080 added to the chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:101 Selenium Manager binary found at: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:114 Executing process: /opt/hostedtoolcache/Python/3.12.10/x64/lib/python3.12/site-packages/selenium/webdriver/common/linux/selenium-manager --browser chrome --debug --language-binding python --output json
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Found chromedriver 137.0.7151.55 in PATH: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 chrome detected at /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Running command: /usr/bin/google-chrome --version
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Output: "Google Chrome 137.0.7151.55 "
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Detected browser: chrome 137.0.7151.55
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Required driver: chromedriver 137.0.7151.68
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Driver path: /usr/bin/chromedriver
DEBUG    selenium.webdriver.common.selenium_manager:selenium_manager.py:140 Browser path: /usr/bin/google-chrome
DEBUG    selenium.webdriver.common.service:service.py:231 Started executable: `/usr/bin/chromedriver` in a child process with pid: 10713 using 0 to output -3
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:35267
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "GET /session/0c18b68b16b994b4f1ca1e606e0837c6/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/element/f.F9CE97554D1D1ADA43093A175A15A937.d.E8EBC29E67878BDECC519CDE484193E7.e.2/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/element/f.F9CE97554D1D1ADA43093A175A15A937.d.E8EBC29E67878BDECC519CDE484193E7.e.3/value HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/element/f.F9CE97554D1D1ADA43093A175A15A937.d.E8EBC29E67878BDECC519CDE484193E7.e.6/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "GET /session/0c18b68b16b994b4f1ca1e606e0837c6/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "GET /session/0c18b68b16b994b4f1ca1e606e0837c6/cookie HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:241 Starting new HTTP connection (1): localhost:8080
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:8080 "GET /crumbIssuer/api/json HTTP/1.1" 200 155
DEBUG    core.jenkins_utils:jenkins_utils.py:31 update_crumb: 87c8962cc221e3d0d0bd91804230e0451168b31987621235c71f51c98f1cc4ac
DEBUG    LoginPage:login_page.py:32 login crumb: 87c8962cc221e3d0d0bd91804230e0451168b31987621235c71f51c98f1cc4ac
DEBUG    MainPage:ui_element.py:59 Click on locator ('link text', 'Manage Jenkins')
DEBUG    MainPage:ui_element.py:28 waiting for element_to_be_clickable(('link text', 'Manage Jenkins'),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "GET /session/0c18b68b16b994b4f1ca1e606e0837c6/element/f.F9CE97554D1D1ADA43093A175A15A937.d.5BB5C0FE6E8116594497042551E1EBB4.e.45/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/element/f.F9CE97554D1D1ADA43093A175A15A937.d.5BB5C0FE6E8116594497042551E1EBB4.e.45/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "GET /session/0c18b68b16b994b4f1ca1e606e0837c6/url HTTP/1.1" 200 0
DEBUG    ManageJenkinsPage:ui_element.py:59 Click on locator ('xpath', "//a[@href='load-statistics']/..")
DEBUG    ManageJenkinsPage:ui_element.py:28 waiting for element_to_be_clickable(('xpath', "//a[@href='load-statistics']/.."),)
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/element HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/execute/sync HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "GET /session/0c18b68b16b994b4f1ca1e606e0837c6/element/f.F9CE97554D1D1ADA43093A175A15A937.d.EE954A3043D6CC7C481CFF7FF5666811.e.80/enabled HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "POST /session/0c18b68b16b994b4f1ca1e606e0837c6/element/f.F9CE97554D1D1ADA43093A175A15A937.d.EE954A3043D6CC7C481CFF7FF5666811.e.80/click HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "GET /session/0c18b68b16b994b4f1ca1e606e0837c6/url HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "GET /session/0c18b68b16b994b4f1ca1e606e0837c6/title HTTP/1.1" 200 0
DEBUG    urllib3.connectionpool:connectionpool.py:544 http://localhost:35267 "DELETE /session/0c18b68b16b994b4f1ca1e606e0837c6 HTTP/1.1" 200 0